src/templates/weekly.html

```diff
@@ -0,0 +1,88 @@
+{%extends 'base2.html'%}
+{% block body %}
+<br>
+{%if user.weekli|length !=0 and count==1 %}
+
+      <div class="float-center">
+        <h3>Hello {{user.firstname}}! Here are Your Weekly Reports! </h3>
+     </div>
+
+  <div>
+  {%for weeks in user.weekli %}
```

Comment:
we dont want to keep the table header in the for loop and we want to add a conditional for the table header if we don't want to see it if there arent any rows of data